### PR TITLE
Update changelog and bump

### PR DESCRIPTION
**Pull request by NicolasDorier**
Still need small review of #5340

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
...
+* Fix: Revert to default block explorer button wasn't working (#5340) @NicolasDorier
+* Fix: Payment request doesn't have a concept of processing status, goes immediately to settled for on-chain payments (#5309, #5297) @dennisreimann
+* NFC: Fix error display (#5305) @dennisreimann
+
+### Improvements
+
+* Email Rules: Add default texts and document placeholders (#5314) @dennisreimann
+* UI: The on-chain addresses should only be truncated on the middle (#5313, #5311) @dennisreimann
+* Allow configuring nfc permission beforehand (#5319) @Kukks
+* Remove legacy confusing export (#5293) @NicolasDorier
```
Comment (dennisreimann) [inline Changelog.md:28]:
```suggestion
* Remove legacy export (#5293) @NicolasDorier
```

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+* Fix: Transient error 500 when accessing the wallet page (#5326, #5328) @NicolasDorier
+* Fix: Revert to default block explorer button wasn't working (#5340) @NicolasDorier
+* Fix: Payment request doesn't have a concept of processing status, goes immediately to settled for on-chain payments (#5309, #5297) @dennisreimann
+* NFC: Fix error display (#5305) @dennisreimann
+
+### Improvements
+
+* Email Rules: Add default texts and document placeholders (#5314) @dennisreimann
+* UI: The on-chain addresses should only be truncated on the middle (#5313, #5311) @dennisreimann
+* Allow configuring nfc permission beforehand (#5319) @Kukks
```
Comment (dennisreimann) [inline Changelog.md:27]:
```suggestion
* Store settings: Allow configuring NFC permission beforehand (#5319) @Kukks
```

File: Changelog.md [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
+* Login: Add Remember Me button (#5307, #5302) @dennisreimann
+* Archive stores and apps (#5296) @dennisreimann
+* New permission: Can archive pull payment (#5274) @Kukks
+* Add Show QR (LNURL-Withdraw) code in view pull payment view (#5274) @Kukks
+
+### Bug fixes
+
+* Fix: Transient error 500 when accessing the wallet page (#5326, #5328) @NicolasDorier
+* Fix: Revert to default block explorer button wasn't working (#5340) @NicolasDorier
+* Fix: Payment request doesn't have a concept of processing status, goes immediately to settled for on-chain payments (#5309, #5297) @dennisreimann
```
Comment (dennisreimann) [inline Changelog.md:20]:
```suggestion
* Payment Request: Reflect processing status for on-chain payments (#5309, #5297) @dennisreimann
```

File: Changelog.md [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+### New feature
+
+* Add payouts report (#5320) @Kukks
+* Wallet: Delete custom labels (#5324, #5237) @dennisreimann
+* Dashboard: Show revenue data for keypad (#5317) @dennisreimann
+* Dashboard: Show the number of paid invoices in the last 7 days (#5316, #5300) @dennisreimann
+* Login: Add Remember Me button (#5307, #5302) @dennisreimann
+* Archive stores and apps (#5296) @dennisreimann
+* New permission: Can archive pull payment (#5274) @Kukks
+* Add Show QR (LNURL-Withdraw) code in view pull payment view (#5274) @Kukks
```
Comment (dennisreimann) [inline Changelog.md:14]:
```suggestion
* Pull Payment: Show QR code for LNURL-Withdraw (#5274) @Kukks
```

File: Changelog.md [hunk 5]
```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## 1.11.5
+
+### New feature
+
+* Add payouts report (#5320) @Kukks
```
Comment (dennisreimann) [inline Changelog.md:7]:
```suggestion
* Reporting: Add payouts (#5320) @Kukks
```

**Review by dennisreimann — APPROVED** [review]

**Comment by ittybittybit** [comment]
what would be the recommendation  update change log and bump '5341

**Pull request merged**